schema/new_library_verification_cases.json

```diff
@@ -438,6 +438,243 @@
+                "weather": "../weather/USA_WA_Seattle-Tacoma.Intl.AP.727930_TMY3.epw",
+                "output": "eplusout.csv",
+                "ep_path": "C:\\EnergyPlusV9-0-1\\energyplus.exe"
+            },
+            "expected_result": "pass",
+            "datapoints_source": {
+                "idf_output_variables": {
+                    "T_wh_inlet": {
+                        "subject": "Core_ZN Water Equipment Water Inlet Node",
+                        "variable": "System Node Temperature",
+                        "frequency": "TimeStep"
+                    }
+                },
+                "parameters": {
+                }
+            },
+            "verification_class": "ServiceWaterHeatingSystemControl"
+        },
+		{
+            "no": 18,
+            "run_simulation": false,
+            "simulation_IO": {
+                "idf": "",
+                "idd": "",
+                "weather": "",
+                "output": "VAVStaticPressureSensorLocation.csv",
+                "ep_path": ""
+            },
+            "expected_result": "pass",
+            "datapoints_source": {
+                "idf_output_variables": {
+					"p_fan_setpoint": {
+                        "subject": "sensor",
+                        "variable": "static_pressure_setpoint",
+                        "frequency": "detailed"
+                    }
+                },
+                "parameters": {
+                }
+            },
+            "verification_class": "VAVStaticPressureSensorLocation"
+        },
+		 {
+            "no": 19,
+            "run_simulation": true,
+            "simulation_IO": {
+                "idf": "../test_cases/doe_prototype_cases/ASHRAE901_OfficeSmall_STD2019_Seattle_case19.idf",
+                "idd": "../resources/Energy+V9_0_1.idd",
+                "weather": "../weather/USA_WA_Seattle-Tacoma.Intl.AP.727930_TMY3.epw",
+                "output": "eplusout.csv",
+                "ep_path": "C:\\EnergyPlusV9-0-1\\energyplus.exe"
+            },
+            "expected_result": "pass",
+            "datapoints_source": {
+                "idf_output_variables": {
+                    "no_of_occ": {
+                        "subject": "Core_ZN",
+                        "variable": "People Occupant Count",
+                        "frequency": "Timestep"
+                    },
+					"Q_load": {
+                        "subject": "Core_ZN",
+                        "variable": "Zone Predicted Sensible Load to Setpoint Heat Transfer Rate",
+                        "frequency": "Timestep"
+                    },
+					"P_fan": {
+                        "subject": "PSZ-AC:1 Fan",
+                        "variable": "Fan Electric Power",
+                        "frequency": "Timestep"
+                    }
+                },
+                "parameters": {
+                }
+            },
+            "verification_class": "VentilationFanControl"
+        },
+		{
+            "no": 20,
```

Comment:
I only see `20` cases here but we have `23` library ids, are we missing some?

Reply:
I don't think numbers in the two files match. For example, in```new_library_verification_cases.json``` file,  there are a total of 4 ```SupplyAirTempReset``` cases. Further, the ```NightCycleOperation```'s number in is 15 ```library.json``` and 13 in ```new_library_verification_cases.json```. Are the numbers supposed to match?

Reply:
Nope, thanks for checking.